README.md

```diff
@@ -7,7 +7,7 @@ Swifty Starters includes the extensions and basic files required for kickstartin
 - [Starter Files](#starter_files)
 
 ## Requirements:
-- iOS 8.0+ / tvOS 9.0+ / watchOS 3.0+ / macOS 10.10+
+- iOS 8.0+
```

Comment:
Since iOS version and swift version are independent, I'm thinking of removing this all together. @leapfrog-saugat Should we?